app/gui/src/presenter/graph/state.rs

```diff
@@ -619,20 +621,18 @@ impl<'a> ControllerChange<'a> {
     }
 
     /// Set the new expression's method pointer. If the method pointer actually changes, the
-    /// to-be-updated view is returned.
+    /// to-be-updated view and widget target is returned.
```

Comment:
1. What is "widget target"?
2. I don't see conenction between "widget target" and `Option<ast::Id>`.